domains/devdaim.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "sYnceDez41",
+    "email": "daimdev6@gmail.com"
+  },
+  "record": {
+    "A": ["4.247.175.26"]
```

Comment:
can you show a preview of your website?

Reply:
![image](https://github.com/user-attachments/assets/cf60edf9-3eea-4a19-ab68-e2ef71834356)
![image](https://github.com/user-attachments/assets/779981b0-1147-4c35-8689-1ad42beceb37)
![image](https://github.com/user-attachments/assets/c1790f23-21cc-43e3-98e1-c3e397e06663)
![image](https://github.com/user-attachments/assets/aa4b7552-d09b-4edc-80bd-d27e0f2a24dc)
![image](https://github.com/user-attachments/assets/df7a9a0a-2fb1-4095-a70e-b69c64041f7e)


ALMOST COMPLETE @Stef-00012